cypress/e2e/iframe.cy.ts

```diff
@@ -0,0 +1,11 @@
+import {IframePage} from "../page/index";
```

Comment:
as this is for the next step, could you deleted and wait until merge the changes related to upload-download page

Reply:
Ok, I've deleted all the stuff from the next step, thx